scripts/README.md

```diff
@@ -1,51 +1,47 @@
+	- `--verbose`: Enable verbose output, default = False
+	- `--plot-uv-analysis`: Enable UV analysis plotting
+	- `--plot-dirty-images`: Enable dirty images plotting
 
-- `run_classifier.sh': This script starts the ML classifier script which data is piped to from stdin.
-- `run_clssifier.py': This script receives image cutouts for NSFRB candidates piped from `run_search.py' and converts them to numpy arrays. [DETAILS TBD]
+- `get_status.sh [wait_time] [num_lines]`: This function reads log files currently in the `dsa110-nsfrb/tmpoutput` and `dsa110-nsfrb/process_server` folders and outputs them to terminal for periodic monitoring. It takes the following command-line arguments:
+	- `wait_time`: time in seconds between updates
+	- `num_lines`: number of lines displayed from the end of each log file
 
-[TODO: ADD ARGUMENT OPTIONS]
+- `SearchAlgorithmTuning_V1.ipynb`: This is a jupyter notebook used for prototyping and testing the search system.
 
-- `script_warnings'
-    - `searchlog_warnings.txt': This records stderr output piped from the `run_search.py' script
-    - `classlog_warnings.txt': This records stderr output piped from the `run_classifier.py' script
+- `get_status.sh`: This function reads log files currently in the `dsa110-nsfrb/tmpoutput` folder and outputs them to terminal for periodic monitoring, taking the wait time between updates in seconds as the argument
 
-- `script_flags'
-    - `searchlog_flags.txt': This records data info about data piped from `run_search.py' to `run_classifier.py' needed to convert raw hex to numpy array. This includes `datasize' (int specifying the total number of bytes output from `run_search.py'), `outputshape' (tuple specifying the shape of the numpy array), and `size' (int specifying the number of bits for each entry in the numpy array); each argument is delimited by a `;'.
-
-- `get_status.sh': This function reads log files currently in the `dsa110-nsfrb/tmpoutput' folder and outputs them to terminal for periodic monitoring, taking the wait time between updates in seconds as the argument
-
-- `socket_client_test_PUT.sh': This script uses curl (https://curl.se/) to take an input datafile and send using an http PUT command to the T4 server. 
-- `socket_client_test_POST.sh' [deprecated] : This script uses curl (https://curl.se/) to take an input datafile and send using an http POST command to the T4 server. This script has been replaced by the more reliable and efficient `socket_client_test_PUT.sh'.
+- `socket_client_test_PUT.sh`: This script uses curl (https://curl.se/) to take an input datafile and send using an http PUT command to the T4 server. 
+- `socket_client_test_POST.sh` [deprecated] : This script uses curl (https://curl.se/) to take an input datafile and send using an http POST command to the T4 server. This script has been replaced by the more reliable and efficient `socket_client_test_PUT.sh`.
 
+- `test_search.py`: This script runs unit tests for the search pipeline.
```

Comment:
@mbsherma remove this b/c testing moved to tests folder

Reply:
Resolved in [78cb0ed](https://github.com/dsa110/dsa110-nsfrb/pull/10/commits/78cb0edfc85ffdf445089667f71141e579d956f9)